site2/docs/administration-isolation.md

```diff
@@ -0,0 +1,84 @@
+---
+id: administration-isolation
+title: Pulsar isolation
+sidebar_label: Pulsar isolation
+---
+
+In an organization, a Pulsar instance provides services to multiple teams. When organizing the resources across multiple teams, you want to make a suitable isolation plan to avoid the resource competition between different teams and applications and provide high-quality messaging service. In this case, you need to take resource isolation into consideration and weigh your intended actions against expected and unexpected consequences.
+
+To enforce resource isolation, you can use the Pulsar isolation policy, which allows you to allocate resources (**broker** and **bookie**) for the namespace.
+
+## Broker isolation
+
+In Pulsar, when namespaces (more specifically, namespace bundles) are assigned dynamically to brokers, the namespace isolation policy limits the set of brokers that can be used for assignment. Before topics are assigned to brokers, you can set the namespace isolation policy with a primary or a secondary regex to select desired brokers.
+
+You can set a namespace isolation policy for a cluster using one of the following methods. 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Admin CLI-->
+
+```
+pulsar-admin ns-isolation-policy set options
+```
+
+For more information about the command and descriptions, see [here](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-set-em-).
```

Comment:
I think it's better not to specify the specific version of pulsar-admin

Reply:
Thanks for your suggestion. While the Pulsar website itself (Pulsar Admin CLI doc) does not contain this command and the information on the "Pulsar Admin CLI" page is far behind from pulsar-admin where contains the latest information.

Seems that there was an issue (create automatic workflow) to sync contents from pulsar-admin to the Pulsar website, but no more updates for a long time. I think right now the best choice is to point users to get the latest information and we can update the link once we have a better option.

Reply:
@Huanli-Meng thanks for your suggestion. While the Pulsar website itself [(Pulsar Admin CLI doc)](https://pulsar.apache.org/docs/en/next/pulsar-admin/#policies) does not contain this command and the information on the "Pulsar Admin CLI" page is far behind from [pulsar-admin](http://pulsar.apache.org/tools/pulsar-admin/) where contains the latest information. 

Seems that there was an issue (create automatic workflow) to sync contents from [pulsar-admin](http://pulsar.apache.org/tools/pulsar-admin/) to the Pulsar website, but no more updates for a long time. I think right now the best choice is to point users to get the latest information and we can update the link once we have a better option.

Reply:
@Anonymitaet I have the same concern here, it's better not to specify the version. You can use the general one http://pulsar.apache.org/tools/pulsar-admin/
or use variable for the version.

Reply:
@Jennifer88huang but http://pulsar.apache.org/tools/pulsar-admin/ does not refer to the specific command and users need to search, which is not convenient for users.

Reply:
OK, I've changed them to general links.

Reply:
then you can use variable, like this
http://pulsar.apache.org/tools/pulsar-admin/{{pulsar:version}}-SNAPSHOT/#-em-set-em-

Reply:
@Jennifer88huang this variable might do not work, since for minor releases, for example, 2.6.3, http://pulsar.apache.org/tools/pulsar-admin/ does not have 2.6.3-SNAPSHOT, so I change the specific link (http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-set-em-) to a general link http://pulsar.apache.org/tools/pulsar-admin/